apps/common/views.py

```diff
@@ -22,3 +23,21 @@ class ProtectedTestView(GenericAPIView):
     @staticmethod
     def get(request: Request) -> Response:
         return Response({"live": True})
+
+
+class AdminSpectacularSwaggerView(SpectacularSwaggerView):
```

Comment:
In drf-spectacular este settings pentru asta si nu trebuie sa creezi views custom